src/translations/en.json

```diff
@@ -4201,6 +4203,15 @@
         "id": "ID",
         "attributes": "Attributes",
         "device_path": "Device path"
+      },
+      "datadisk_move": {
+        "title": "[%key:supervisor::system::host::move_datadisk%]",
+        "description": "You are currently using ''{current_path}'' as datadisk, moving it will reboot your device.",
+        "select_device": "Select new datadisk",
+        "no_devices": "No suitable attached devices found",
+        "confirm_text": "Do you want to move it?",
```

Comment:
Is that key used somewhere and I am just not seeing it?

Reply:
```suggestion
```